src/interfaces/business-i.ts

```diff
@@ -87,3 +87,45 @@ export interface BusinessI extends SlimBusinessI {
   submitter?: string // not used
   warnings: Array<BusinessWarningI>
 }
+
+export const getDefaultEmptyBusiness = (): BusinessI => {
```

Comment:
I don't think this should be inside the interfaces and it isn't being used by anything can you remove ?